src/math/p5.Vector.js

```diff
@@ -3887,5 +3887,34 @@ p5.Vector = class {
     }
     return v.equals(v2);
   }
-};
-export default p5.Vector;
+
+
+  /**
+ * ClampToZero check against a <a href="#/p5.Vector">p5.Vector</a>
```

Comment:
Can I suggest:

> Replaces the components of a `p5.Vector` that are <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON">very close to zero</a> with zero.
>
> Sometimes, math in JavaScript can give in slightly imprecise answers due to the way numbers are represented. This can make it hard to check if a number is zero, as it may be close but not exactly zero. This method rounds very close numbers to zero to make those checks easier.

Reply:
Wow this is so much more clearer

Can I change this part 

> Sometimes, math in JavaScript can give in

to this 

> In computers handling numbers with decimals can give

So the new text will become -

> In computers handling numbers with decimals can give slightly imprecise answers due to the way those numbers are represented. This can make it hard to check if a number is zero, as it may be close but not exactly zero. This method rounds very close numbers to zero to make those checks easier

Reply:
Sounds good! I'd maybe just add a comma after "In computers".